### PR TITLE
Revert unsuccessful changes, tune Learning rate and capacity

### PR DESCRIPTION
**Pull request by Marko-Shevchuk**

### CODE REVIEW

**Comment by Marko-Shevchuk** [comment]
Unsuccessful changes shouldn't be reverted: the best trial should simply become the center of the hyperparam space
